images/Makefile

```diff
@@ -31,10 +31,12 @@ push-all: build-all
 
 build-all:
 	$(MAKE) ${OP} WHAT=amazon-kernel
-	$(MAKE) ${OP} WHAT=amazonlinux   RELEASE=2     IS_LATEST=true
+	$(MAKE) ${OP} WHAT=amazonlinux   RELEASE=2          IS_LATEST=true
 	$(MAKE) ${OP} WHAT=alpine
+	$(MAKE) ${OP} WHAT=opensuse      RELEASE=leap
+	$(MAKE) ${OP} WHAT=opensuse      RELEASE=tumbleweed IS_LATEST=true
```

Comment:
We use `IS_LATEST` to signal which image gets the `:latest` tag, not necessarily which image is the latest _release_ (see the Ubuntu image). For me it seems more logical to have the stable leap release cycle as the default (`:latest`), unless it's more common to run tumbleweed in OpenSUSE land.